SonarQube.Client/Api/V5_10/GetIssuesRequest.cs

```diff
@@ -88,6 +88,6 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
 
         private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
             new SonarQubeIssue(issue.Path, issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey, issue.RuleKey,
-                issue.Status == "RESOLVED");
+                issue.Status == "RESOLVED", flows: null);
```

Comment:
This request class handles very old SQ instances. In that version the response was returned as a serialized protobuf stream. I tried installing a SQ5.1 instance so I could generate the expected protobuf response for the test, but that version of SQ is so old that it needs JRE v2 so I decided it wasn't worth it.

Instead, I added a test at the model level to make sure the flows collection is never null in the returned read-only data classes.